src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -547,51 +540,25 @@ public void setPluginManager(PluginManager pluginManager) {
      * that we need for testing.
      */
     protected ServletContext createWebServer2() throws Exception {
-        QueuedThreadPool qtp = new QueuedThreadPool();
```

Comment:
All of the deleted code in this file was previously duplicated with `JenkinsRule`. It is now consolidated in:

- `JettyProvider`
- `Jetty12EE9Provider`
- `Jetty12EE10Provider`